### PR TITLE
Ongoing Monitoring

### PR DESCRIPTION
**Pull request by tomer-shvadron**
### Description
* Fixed unused `projectIds` in `FilterRepository`
* Added `config` column to `Customer` table
* Created new `Webhook` module, controller and service
* Added an endpoint for AML webhooks (supports `entityType = 'individuals'` only for now)
* Some boyscouting

### CODE REVIEW

File: services/workflows-service/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "db:migrate-up": "prisma migrate deploy",
     "db:reset": "prisma migrate reset --skip-seed -f",
     "db:reset:dev": "npm run db:reset && npm run seed",
-    "db:reset:dev:with-data": "npm run db:reset && npm run seed && npm run db:data-migration:migrate",
+    "db:reset:dev:with-data": "npm run db:reset:dev && npm run db:data-migration:migrate",
```
Comment (tomer-shvadron) [inline services/workflows-service/package.json]: Boyscouting

File: services/workflows-service/src/filter/filter.repository.ts [hunk 2]
```diff
@@ -27,10 +27,13 @@ export class FilterRepository {
...
-      this.scopeService.scopeFindFirst({
-        ...args,
-        where: { ...args?.where, id: id },
-      }),
+      this.scopeService.scopeFindFirst(
+        {
+          ...args,
+          where: { ...args?.where, id: id },
+        },
+        projectIds,
```
Comment (tomer-shvadron) [inline services/workflows-service/src/filter/filter.repository.ts]: Boyscouting. We didn't use `projectIds` for some reason, even though it's not an unscoped function

File: services/workflows-service/src/workflow-defintion/workflow-definition.service.ts [hunk 3]
```diff
@@ -80,12 +80,8 @@ export class WorkflowDefinitionService {
 
   async getLatestVersion(id: string, projectIds: TProjectIds) {
     const workflowDefinition = await this.repository.findById(id, {}, projectIds);
-    const latestVersion = await this.repository.findByLatestVersion(
-      workflowDefinition.name,
-      projectIds,
-    );
 
-    return latestVersion;
+    return await this.repository.findByLatestVersion(workflowDefinition.name, projectIds);
```
Comment (tomer-shvadron) [inline services/workflows-service/src/workflow-defintion/workflow-definition.service.ts]: Boyscouting

File: services/workflows-service/src/workflow/workflow.service.ts [hunk 4]
```diff
@@ -85,27 +85,8 @@ import { ajv } from '@/common/ajv/ajv.validator';
 
 type TEntityId = string;
 
-export const ResubmissionReason = {
```
Comment (tomer-shvadron) [inline services/workflows-service/src/workflow/workflow.service.ts]: Boyscouting. Were unused

**Review by Omri-Levy — APPROVED** [review]

**Pull request merged**